cypress/integration/version/task_filters.ts

```diff
@@ -142,19 +146,23 @@ describe("Tasks filters", () => {
 
   describe(`Task Base Statuses select`, () => {
     const urlParam = "baseStatuses";
+    before(() => {
+      cy.visit(pathURLWithFilters);
```

Comment:
this is a hack to make it easier to select the antd table dropdown. clicking on the filter dropdown permanently appends additional html elements to the dom even after closing.

Reply:
Checkbox selection utilizing the id attribute for a label to search for the input fails to get the correct Id for the checkbox after an antd dropdown is initially rendered. This likely has to do with way antd injects additional elements into the page after triggering a table dropdown. I think it's okay to keep the hard reset since we will switch over to the leafygreen table where we will have control over how/when the dropdown element attaches to the dom.